libs/map/framework.cpp

```diff
@@ -3339,15 +3339,15 @@ std::optional<products::ProductsConfig> Framework::GetProductsConfiguration() co
 void Framework::DidCloseProductsPopup(ProductsPopupCloseReason reason) const
 {
   settings::Set(kPlacePageProductsPopupCloseTime, base::SecondsSinceEpoch());
-  settings::Set(kPlacePageProductsPopupCloseReason, std::string(ToString(reason)));
+  settings::Set(kPlacePageProductsPopupCloseReason, ToString(reason));
 }
 
 void Framework::DidSelectProduct(products::ProductsConfig::Product const & product) const
 {
   settings::Set(kPlacePageSelectedProduct, product.GetTitle());
 }
 
-uint32_t Framework::GetTimeoutForReason(ProductsPopupCloseReason reason) const
+uint32_t Framework::GetTimeoutForReason(ProductsPopupCloseReason reason)
```

Comment:
Why is it in the framework? Can be in a separate file or in product-related file.